notebooks/117-model-server/117-model-server.ipynb

```diff
@@ -0,0 +1,549 @@
+    "Model Server hosts models and makes them accessible to software components over standard network protocols: a client sends a request to the model server, which performs model inference and sends a response back to the client. Model Server offers many advantages for efficient model deployment:\n",
+    "\n",
+    "- Remote inference enables using lightweight clients with only the necessary functions to perform API calls to edge or cloud deployments.\n",
+    "- Applications are independent of the model framework, hardware device, and infrastructure.\n",
+    "- Client applications in any programming language that supports REST or gRPC calls can be used to run inference remotely on the model server.\n",
+    "- Clients require fewer updates since client libraries change very rarely.\n",
+    "- Model topology and weights are not exposed directly to client applications, making it easier to control access to the model.\n",
+    "- Ideal architecture for microservices-based applications and deployments in cloud environments – including Kubernetes and OpenShift clusters.\n",
+    "- Efficient resource utilization with horizontal and vertical inference scaling.\n",
+    "  \n",
+    "![ovms_diagram](https://user-images.githubusercontent.com/91237924/215658773-4720df00-3b95-4a84-85a2-40f06138e914.png)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "efce7a1c",
+   "metadata": {},
+   "source": [
+    "## Serving with OpenVINO Model Server\n",
+    "OpenVINO Model Server (OVMS) is a high-performance system for serving models. Implemented in C++ for scalability and optimized for deployment on Intel architectures, the model server uses the same architecture and API as TensorFlow Serving and KServe while applying OpenVINO for inference execution. Inference service is provided via gRPC or REST API, making deploying new algorithms and AI experiments easy.\n",
+    "\n",
+    "![ovms_high_level](https://user-images.githubusercontent.com/91237924/215658767-0e0fc221-aed0-4db1-9a82-6be55f244dba.png)\n",
+    "\n",
+    "To quickly start using OpenVINO™ Model Server follow these steps:"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "740bfdd8",
+   "metadata": {},
+   "source": [
+    "## Step 1: Prepare Docker\n",
+    "Install [Docker Engine](https://docs.docker.com/engine/install/), including its [post-installation](https://docs.docker.com/engine/install/linux-postinstall/) steps, on your development system. To verify installation, test it using the following command. If it displays a test image and a message, it is ready."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "73d7aedb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!docker run hello-world"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "85b48949",
+   "metadata": {},
+   "source": [
+    "## Step 2: Download the Model Server\n",
```

Comment:
If we would like to reduce the number of steps, this one could be dropped. OVMS image will be pulled with `docker run` command in step 4, if it's not on the machine already.

Reply:
done